src/components/Button/index.jsx

```diff
@@ -2,15 +2,21 @@ import React from "react";
 import { Link } from "react-router-dom";
 import styles from "./styles/styles.module.scss";
 
-const Button = ({ path, type, children }) => {
+const Button = ({ path, type, children, isFormValid }) => {
   let buttonStyle = styles.button;
+  let nextPath = path;
```

Comment:
```suggestion
  let newPath = path;
```
Because `Button` is a generic component which can be used in many places.